docs/production-deployment/cloud/billing-and-cost.mdx

```diff
@@ -0,0 +1,77 @@
+---
+id: billing-and-cost
+title: Billing and Cost
+sidebar_label: Billing and cost
+sidebar_position: 3
+description: As an Account Owner, you can access and manage billing details anytime, with invoices available for download on the Billing page. The typical billing cycle begins on the first of the month (UTC).
+slug: /cloud/billing-and-cost
+toc_max_heading_level: 4
+keywords:
+  - billing
+  - invoices
+  - cost management
+  - account owner
+  - billing cycle
+  - payment options
+  - credit usage
+  - temporal cloud
+  - account balance
+  - invoice history
+tags:
+  - billing
+  - invoices
+  - cost management
+  - account owner
+  - billing cycle
+  - payment options
+  - credit usage
+  - temporal cloud
+  - account balance
+  - invoice history
+---
+
+The Account Owner holds the ability to access and view detailed billing and cost information at any time, ensuring full transparency and control over financial aspects. If you need to assess your spending patterns, check invoice histories, or handle payment details, Temporal Cloud's Cost and Billing capabilities are able to provide the information you need.
+
+## About the Billing page
```

Comment:
I don't think you neccessarily need this title. We also don't generally start the title with About or How

Reply:
Alternatively, you can title it: 
H2: ## Accessing Billing information in Temporal Cloud
H3: ### Navigate to your invoices
H3: ### Current Balance 
H3: ### Recent Bill
H3: ### Invoice Table